src/selectors/entities/common.js

```diff
@@ -4,6 +4,7 @@
 
 import type {GlobalState} from '../../types/store';
 import type {UserProfile} from '../../types/users';
+import type {ChannelMembership} from '../../types/channels';
```

Comment:
Please change all these to be fullPath instead

Reply:
I know is not the scope of the PR but why not do it, right?

Reply:
Yes, the reason behind that is when you use it in other repositories, like in mattermost-webapp you find correctly it, if we use absolute path imports, it wont be found.

Reply:
Why?

Reply:
https://github.com/facebook/flow/issues/4186

Reply:
Yes, is a common problem. While I'm developing I can use the global reference, but the types sharing with other library is basically "copy" the file to export the types, so the absolute path in the other project doesn't work. I going to try to do something like "resolve locally mattermost-redux/whatever to src/whatever", allowing all imports use mattermost-redux/types/store. That would work for you?

Reply:
I uploaded a commit with the change, I have to test it in webapp flow changes, to verify that everything is working.